src/client/components/Groups/View/GroupViewContainer.component.js

```diff
@@ -36,13 +36,36 @@ import './scss/groupView.scss';
 export class GroupViewContainer extends React.Component {
   constructor(props) {
     super(props);
+
+    this.defaultMembersLoadLimit = 9; // first view is 9 members including the owner
+    this.maxMembersReturned = 8; // we don't want more than 8 memebers for default display
+
     this.state = {
       following: props.group.isFollowing,
       showloginToFollowMessage: false
     };
 
     this.toggleFollow = this.toggleFollow.bind(this);
-    this.toggleMembersExpanded = this.toggleMembersExpanded.bind(this);
+  }
+
+  componentWillMount() {
+    const {membersCount} = this.props.group;
+    let limit = null;
+    let offset = null;
+    if (membersCount > this.defaultMembersLoadLimit) {
+      limit = this.defaultMembersLoadLimit;
+      offset = Math.round(Math.random() * (membersCount - limit));
+    }
+
+    this.props.groupActions.asyncGetGroupMembers(this.props.group.id, [this.props.group.owner.id], limit, offset, this.maxMembersReturned);
+  }
+
+  loadAllMembers() {
+    if (this.props.group.membersCount > this.defaultMembersLoadLimit) {
+      const excludedIds = this.props.group.members.map(m => m.id);
```

Comment:
Måske en kommentar til hvorfor vi har excludedIds. Er det for at undgå at hente de første 9 medlemmer ud igen?